src/apps/tco2_dashboard/onchain_pool_comp.py

```diff
@@ -3,55 +3,144 @@
 import dash_bootstrap_components as dbc
 
 
-def create_onchain_pool_comp_content(token_cg_dict, df_prices, fig_historical_prices):
+def create_onchain_pool_comp_content(tokens_dict, df_prices, fig_historical_prices):
 
+    historical_price_note = "Note: The chart shows MCO2 prices based on the KLIMA/MCO2 pool since it launched."
     pool_cards = []
-    for i in token_cg_dict.keys():
+    for i in tokens_dict.keys():
+        filtered_df = df_prices[~df_prices[f"{i}_Price"].isna()]
+        bridge_name = tokens_dict[i]["Bridge"]
+        if i == "MCO2":
+            selective_cost_value = "NA"
+            selective_cost_tooltip_text = (
+                "There is no selective reddeeming/retiring option for MCO2."
```

Comment:
Typo here, should be "redeeming"

Would also reword to "There is no selective redemption/retirement functionality for MCO2."

Reply:
Made the change; in the following commit.